<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>screen_saver</class>
 <widget class="QDialog" name="screen_saver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#screen_saver
{
background-color: transparent;
}</string>
  </property>
  <widget class="QLabel" name="label_hour">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>529</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_hour
{
color: rgb(0, 167, 250);
}</string>
   </property>
   <property name="text">
    <string>11:45</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_day">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_day
{
color: rgb(17, 110, 141);
}</string>
   </property>
   <property name="text">
    <string>25</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="notification">
   <property name="geometry">
    <rect>
     <x>684</x>
     <y>33</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#notification
{
color: black;
border: none;
background-color:transparent;
}</string>
   </property>
   <property name="text">
    <string>13</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#background
{
background-image: url(:/iconos/screen800x600/gifs/PTAR-VAB.gif);
background-position: center;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bell">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#bell
{
/*background-image: url(:/iconos/screen800x600/gifs/OK.gif);*/
background-image: url(:/iconos/screen800x600/iconos/Bell_2.png);
background-position: left;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_month">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>490</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_month
{
color: rgb(0, 167, 250);
}</string>
   </property>
   <property name="text">
    <string>Noviembre</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_am_pm">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>533</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_am_pm
{
color: rgb(17, 110, 141);
}</string>
   </property>
   <property name="text">
    <string>PM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_seconds">
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>551</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_seconds
{
color: rgb(17, 110, 141);
}</string>
   </property>
   <property name="text">
    <string>32</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>label_am_pm</zorder>
  <zorder>label_seconds</zorder>
  <zorder>label_hour</zorder>
  <zorder>label_day</zorder>
  <zorder>bell</zorder>
  <zorder>label_month</zorder>
  <zorder>notification</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
